src/xcode/ENA/ENA/Source/HTTPClient/Service/Models/Receive/ValidationRulesModel.swift

```diff
@@ -0,0 +1,25 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+import HealthCertificateToolkit
+import class CertLogic.Rule
+
+struct ValidationRulesModel: CBORDecoding {
+
+	init(decodeCBOR: Data) throws {
```

Comment:
Throwing the error here, looses its type safety. After this call it can be any error and the handling is more complicated. Is there any way to use a typesafe result type?